src/macros.rs

```diff
@@ -0,0 +1,35 @@
+/// Create and populate a mount.
+///
+/// ```ignore
+/// let router = router!("/"       => index,
+///                      "/:query" => queryHandler)
+/// ```
+///
+/// Is equivalent to:
+///
+/// ```ignore
+///    let mut mount = Mount::new();
+///    mount.mount("/", index);
+///    mount.mount("/:query", queryHandler);
```

Comment:
Hmm, this example suggests that mount can deal with variables in paths when it only matches by prefix. Also your macro example says `router!`, not `mount!`.